### ISSUE TITLE
opml file works but there is no content in the corresponding directory

### ISSUE THREAD

**Issue by keiem** [raptor2101/PodCatcher#13]
Hello,

I created this opml file wit a rss feed -> https://pastebin.com/ikGPTzpR.  
The opml file doesn't give a error, I can browse the created map structure, but there is no content to play.  Is there something wrong with my opml or is there a problem in podcatcher as it is marked broken in the kodi repository.

**Comment by raptor2101** [comment]
i'am currently looking into this

**Comment by raptor2101** [comment]
wich version did you use?

opening the OPML with 0.2.1 results in this:

![image](https://user-images.githubusercontent.com/505563/73302331-c16ac300-4214-11ea-87f2-166241d186bc.png)

**Comment by keiem** [comment]
I used master, If I remember I tried 0.2.1 also with the same result.  I
will try again.

thanks for answering.

Op di 28 jan. 2020 om 21:26 schreef Christian Kölpin <
notifications@github.com>:

**Comment by keiem** [comment]
So I reinstalled 0.2.1 but it stays the same.  Maybe because I'm on
LibreElec.

Op di 28 jan. 2020 om 21:59 schreef Roel Vanraepenbusch <roeleman@gmail.com

**Comment by keiem** [comment]
OK, it works.  I removed the folder in addon_data.  Removed the add-on via the add-on manager.  Wanted to install the add-on from the kodi add-on repository, but it doesn't exist there.  So I downloaded it from the kodi mirror for Leia -> https://mirrors.kodi.tv/addons/leia/plugin.audio.podcatcher/
Installed it, run the addon (which is marked as defect in kodi).  Now the folder plugin.audio.podcatcher exist in addon_data.  Put the opml.xml file in it -> result: it works.

What I did the first time: Couldn't find it in the kodi repository, so I downloaded it from your github page.  Installed it as zip. Couldn't find the plugin.audio.podcatcher folder so I created it myself.  Tried my first opml.xml and it didn't work.  Found out there where 2 podcatcher folders now in addon_data.  I removed the one I created.  Tested a dozen of opml.xml files until I found out the one I mention in my first post. Noticed there was no content.  Tried a dozen variants of the opml.xml + installing reinstalling...  What brought me here.  
I don't know what was the wrong step.  The github download or creating the plugin.audio.podcatcher folder.
Nevertheless, it works now.  Thanks to you who pointed out the opml.xml file was OK and it worked with you.
